src/OrbitGl/CGroupAndProcessMemoryTrack.cpp

```diff
@@ -125,40 +125,33 @@ std::string CGroupAndProcessMemoryTrack::GetValueUpperBoundTooltip() const {
       kGameCGroupName, kGameCGroupLimitGB);
 }
 
-void CGroupAndProcessMemoryTrack::OnTimer(const orbit_client_protos::TimerInfo& timer_info) {
-  int64_t cgroup_limit_bytes = orbit_api::Decode<int64_t>(timer_info.registers(static_cast<size_t>(
-      CaptureEventProcessor::CGroupAndProcessMemoryUsageEncodingIndex::kCGroupLimitBytes)));
-  int64_t cgroup_rss_bytes = orbit_api::Decode<int64_t>(timer_info.registers(static_cast<size_t>(
-      CaptureEventProcessor::CGroupAndProcessMemoryUsageEncodingIndex::kCGroupRssBytes)));
-  int64_t cgroup_mapped_file_bytes = orbit_api::Decode<
-      int64_t>(timer_info.registers(static_cast<size_t>(
-      CaptureEventProcessor::CGroupAndProcessMemoryUsageEncodingIndex::kCGroupMappedFileBytes)));
-  int64_t process_rss_anon_kb = orbit_api::Decode<int64_t>(timer_info.registers(static_cast<size_t>(
-      CaptureEventProcessor::CGroupAndProcessMemoryUsageEncodingIndex::kProcessRssAnonKb)));
-
-  if (cgroup_limit_bytes == kMissingInfo || cgroup_rss_bytes == kMissingInfo ||
-      cgroup_mapped_file_bytes == kMissingInfo || process_rss_anon_kb == kMissingInfo) {
+void CGroupAndProcessMemoryTrack::OnCGroupAndProcessMemoryInfo(
+    const orbit_client_data::CGroupAndProcessMemoryInfo& cgroup_and_process_memory_info) {
+  if (cgroup_and_process_memory_info.cgroup_limit_bytes == kMissingInfo ||
```

Comment:
`kMissingInfo` should also be moved to `ClientData` as it's also only used for memory info. You can probably make a copy for now. And it should get a more specific name, hinting at the fact that it's use for structs related to memory info.